src/toast/styled-components.js

```diff
@@ -139,6 +139,10 @@ export const Body = styled<SharedStylePropsArgT>('div', props => {
     transitionTimingFunction: $theme.animation.easeInOutCurve,
     display: 'flex',
     justifyContent: 'space-between',
+    ':focus': {
```

Comment:
This will be visible even when not using keyboard right? It might be worth to use the same approach as in other components - detecting `isFocusVisible`